core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockletDataMap.java

```diff
@@ -196,6 +201,10 @@ private DataMapRowImpl loadToUnsafe(DataFileFooter fileFooter,
           unsafeMemorySummaryDMStore.getSchema()[taskMinMaxOrdinal], maxValues,
           TASK_MAX_VALUES_INDEX, false);
       ordinal++;
+      taskMinMaxOrdinal++;
+
+      summaryRow.setByteArray(minMaxIndex.getNullValues().toByteArray(), taskMinMaxOrdinal);
```

Comment:
We need to calculate null bits based on whole blocks and then set to the summary

Reply:
Added new routine addTaskNullValues which is going to OR all the Null Bitsets from all the blocklets and place the final value into the summary row. Like the one done for minMax.